stdci.yaml

```diff
@@ -6,7 +6,11 @@ sub-stages:
       timeout: 4h
   - k8s-1.11.0-release:
       timeout: 4h
-  - k8s-multus-1.11.1-release
+  - k8s-multus-1.11.1-release:
+      runtime_requirements:
```

Comment:
Looking at: https://ovirt-infra-docs.readthedocs.io/en/latest/CI/STDCI-Configuration/index.html#runtime-requirements
this should be: runtime-requirements